CHANGES.md

```diff
@@ -6,7 +6,30 @@ list](https://github.com/trinodb/trino-python-client/tags), the
 [README](https://github.com/trinodb/trino-python-client/blob/master/README.md)
 and the [PyPI page](https://pypi.org/project/trino/).
 
-## Trino Python client 0.316.0
+## Release 0.317.0
+
+* Add support for creating tables containing `JSON` columns and reading and
+  writing to them with SQLAlchemy.
+  ([#194](https://trinodb/trino-python-client/issues/194))
+* Properly propagate query failures to the client when using `fetchone`.
```

Comment:
I'd generally word this as "Fix query failures not being propagated to the client when..."

I like to start with the words Add/Fix/Improve as much as possible so it's abundantly clear what each type of change is.